.github/workflows/end2end_test.yml

```diff
@@ -267,37 +278,88 @@ jobs:
         run: npx playwright test --reporter=html --trace=on --project=general
         timeout-minutes: 10
       
-      ### Cleanup steps for the workflow ###
-      - name: Show server logs after failure
+      ### Cleanup steps ###
+      - &cleanup-show-server-logs
+        name: Show server logs after failure
         if: failure()
         working-directory: lightly_studio
         run: |
           echo "Server logs:"
           cat server.log
 
-      - name: Upload Playwright report after failure
+      - &cleanup-upload-playwright-report
+        name: Upload Playwright report after failure
         uses: actions/upload-artifact@v4
         if: failure()
         with:
           name: Upload Playwright Report
-          path: lightly_studio_view/playwright-report
+          path: lightly_studio_view/playwright-report-${{ github.job }}
```

Comment:
This step is reused (see YAML anchor). Added job id to upload path to be able to upload both failures.